src/utils/response.ts

```diff
@@ -26,7 +26,7 @@ export function sendRedirect (event: CompatibilityEvent, location: string, code
   event.res.statusCode = code
   event.res.setHeader('Location', location)
   // minimal html document that redirects on client side
-  const html = `<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=${encodeURI(location)}"></head><body></body></html>`
+  const html = `<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=${encodeURI(location)}"></head><body><noscript>Redirecting to <a href=${JSON.stringify(location)}>${location}</a></noscript></body></html>`
```

Comment:
I think meta also works in case of scripts being disabled :) What i meant was to preserve current behavior of display redirect link while navigation happens (instead of blank screen)

Reply:
yes, I think it does too. I was thinking more of a search engine. I still think it's a worse UX to have it flash on screen (because it's _meant_ to be an SSR redirect, and at least the white screen suggests it's 'still loading'), but will update.